perllib/FixMyStreet/SendReport/Angus.pm

```diff
@@ -0,0 +1,146 @@
+package FixMyStreet::SendReport::Angus;
+
+use Moo;
+
+BEGIN { extends 'FixMyStreet::SendReport'; }
+
+use Try::Tiny;
+use Encode;
+use mySociety::Web qw(ent);
+
+sub get_auth_token {
+    my ($self, $authresult) = @_;
+    if ($authresult =~ m/<AuthenticateResult>\s*([^\s]+)\s*<\/AuthenticateResult>/) {
```

Comment:
As XML::Simple is used elsewhere in the file (needs a `use` at the top?), it might be a bit nicer to use it to get the auth token/external ID? Though probably throws in its own issues with external XML...!